### ISSUE TITLE
Always converges to nearly 22dB

### ISSUE THREAD

**Issue by FrancoisMoreau** [mgharbi/hdrnet_legacy#18]
Hi, 

I used your code for training, try do verify its feasibility. 
According to your paper, I used the daseset of FiveK to get the effects of human-retouch. 
The graphic card I used was TITAN Xp, and I trained it for 5 days, however, the loss didn't decrease and the output PSNR value didn't increase. I became stable around 22db.

I tried for several time, for example, from ExperA to ExpertE, B to C, but it always converges to 22dB. When I do the inference part, it didn't perform well as the sky can be colorful sometime. 
Is there anything wrong with this? 
Thank You !

**Comment by mgharbi** [comment]
Dear Francois, 

I do not recall the exact PSNR numbers for the Experts task, but it is expected to be much lower than the other filters, since these are human retouches, and not as consistent as an algorithmic filter. Our paper reports the error in lab space, to be consistent with previous work. ~22 sounds reasonable.

The inference should not show the artifacts you mention however. Do you train at full-resolution? What I found helpful for this task (to save some time), was to specify the target at a lower-resolution for pre-training. Any luck with the pretrained models?

**Comment by dex1990** [comment]
Hi @FrancoisMoreau,
   Have you fixed the problem?
   I have trained the model with my own dataset, when i run the inference part, the sky become colorful like this:
   
![out-1](https://user-images.githubusercontent.com/2503734/38181588-4ca3a3a4-3666-11e8-856a-388d5a176826.png)

input:
![in-1](https://user-images.githubusercontent.com/2503734/38181630-838705be-3666-11e8-8a2b-826debd6515e.png)

**Comment by dex1990** [comment]
Hi @FrancoisMoreau,
Have you fixed the problem?
I have trained the model with my own dataset, when i run the inference part, the sky become colorful like this:
  
![out](https://user-images.githubusercontent.com/2503734/38181667-c7b89bc6-3666-11e8-8ba2-a3cf29819821.png)

input:
![in](https://user-images.githubusercontent.com/2503734/38181693-dff3933a-3666-11e8-8d6a-26316643f68f.png)
@mgharbi ,any suggestion？
Thanks a lot!

**Comment by benjaminlinken** [comment]
Hi @dex1990 
Have you fixed your problem?

**Comment by butterl** [comment]
Hi, @dex1990

Do you got higher PSNR to meet a better image output?

**Comment by lhwcv** [comment]
@dex1990 Have you fixed your problem? I also meet the similar problem when fitting a BW filter.   @mgharbi Any suggestions? Thanks a lot!    ( loss got to ~ 0.003 and PSNR: 30+)  
input:
![input image](https://user-images.githubusercontent.com/15154327/55327671-2bafbb80-54bd-11e9-85cc-541a46790808.jpg)
predicted:
![predicted](https://user-images.githubusercontent.com/15154327/55327792-77626500-54bd-11e9-9e6a-ac749df0a152.png)
GT:
![GT](https://user-images.githubusercontent.com/15154327/55327849-95c86080-54bd-11e9-8f4a-b2dc34e37705.jpg)

**Comment by MKFMIKU** [comment]
@mgharbi @dex1990 I meet the same problem that the sky in results is colorful

![img_1988](https://user-images.githubusercontent.com/13622651/56554657-fe8a9080-65c4-11e9-8ab6-d8b3cdfd31f7.png)

**Comment by tzhong518** [comment]
Hi, @MKFMIKU , @lhwcv 
Have you managed solve the colorful sky problem?
